### ISSUE TITLE
why Vertical tab not  hide in fullscreen mode

### ISSUE THREAD

**Issue by kr-abhi55** [brave/brave-browser#31082]
## Description 

i'm using vertical tab features of brave and i notice that while in fullscreen mode vertical tab don't hide

## Steps to Reproduce

   1. just enable vertical tab features of brave
   2. open youtube or spotify 
   3. start watching in fullscreen mode

## Actual result:

![image](https://github.com/brave/brave-browser/assets/118665057/053fd7db-ce12-4217-9c08-074f9bf74586)

## Brave version (brave://version info)

1.52.122 Chromium: 114.0.5735.110 (Official Build) (64-bit) 

## Version/Channel Information:

- Can you reproduce this issue with the current release? 
- Can you reproduce this issue with the beta channel? 
- Can you reproduce this issue with the nightly channel? 

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? 
- Does the issue resolve itself when disabling Brave Rewards? 
- Is the issue reproducible on the latest version of Chrome?

**Comment by rebron** [comment]
Hi @kr-abhi55 What operating system are using?

**Comment by kr-abhi55** [comment]
Operating System: Debian GNU/Linux 11
KDE Plasma Version: 5.20.5
KDE Frameworks Version: 5.78.0
Qt Version: 5.15.2
Kernel Version: 5.10.0-23-amd64
OS Type: 64-bit
Processors: 4 × Intel® Core™ i3-4005U CPU @ 1.70GHz
Memory: 7.7 GiB of RAM
Graphics Processor: Mesa DRI Intel® HD Graphics 4400

**Comment by onlycitrus** [comment]
Same issue for me on MacOS Ventura

**Comment by kr-abhi55** [comment]
@onlycitrus i may be a bug

**Comment by kr-abhi55** [comment]
> Same issue for me on MacOS Ventura but happen only when the app was in full screen ie when I use the green fullscreen button and then proceed to fullscreen the video. It doesn't happen when the browser is in windowed mode.

it may be a bug

**Comment by Brave-Matt** [comment]
Other users on Community seeing this as well:
https://community.brave.com/t/vertical-tabs-area-remains-when-using-youtube-full-screen/492272/4

This user left a recording of the behavior as well as instructions on a "workaround" (while it does literally work around the issue it is a hassle as well and not viable for long term):
https://youtu.be/WiJiu9m3AAE

> _To remove the issue, put the video in full screen, create a new window, go to the new window (while the video is fullscreen), collapse the tabs in the new window, go back to your fullscreen window where the vertical tab spaces is magically gone.
> To revert the workaround, just hover the mouse over the tabs and it’ll revert to having the ugly space there._

**Comment by Blufords** [comment]
Same issue on Arch Linux, `[Version 1.52.130 Chromium: 114.0.5735.198 (Official Build) (64-bit)]`
I see other issues about this from earlier this year claiming it was fixed, but it seems the issue persists.

**Comment by hho** [comment]
(K)ubuntu `22.04.2 LTS`
Brave `Version 1.57.1 Chromium: 115.0.5790.56 (Official Build) beta (64-bit)`
With the Beta, it seems to be a little better.

For me, fullscreen with vertical tabs:
* works correctly when initiated via the website (e.g. click fullscreen on YouTube or in Google Meet).
* doesn't work, when just hitting F11 – the tab bar completely remains on screen.

**Comment by Brave-Matt** [comment]
https://community.brave.com/t/vertical-tabs-should-not-be-visible-in-fullscreen-mode/492019/7

**Comment by srirambv** [comment]
FWIW I don't think its the vertical tabs, but its the sidebar which is in always show setting that is causing the issue. @rebron this is an issue on all platforms. 

I have vertical tabs and sidebar enabled but only see the sidebar show up when on full screen
![image](https://github.com/brave/brave-browser/assets/17010094/4c9add29-55b8-4a51-aacd-e406b3ce43c5)

**Comment by phocean** [comment]
@srirambv I don't think so, at least on Mac OS. There is no correlation with the sidebar setting : it shows an empty bar in full screen either in "never", "always" or "scroll over" modes.

And it has the issue only when using vertical tabs.

**Comment by 1chappie** [comment]
Same issues here on MacOS, Brave build: Version 1.56.14 Chromium: 115.0.5790.114 (Official Build) (arm64)

**Comment by bilaharipg** [comment]
@1chappie Update your browser. This issue was resolved in the latest update, and you are not using the latest version

**Comment by JavierLuna** [comment]
Confirmed, it solved the issue for me!

**Comment by phildufresne** [comment]
Is there an option to go back to the old behaviour? I like leaving my browser fullscreen on its own virtual desktop, and now I can't do that because my tabs disappear.

**Comment by gmann42** [comment]
Yeah I also want the vertical side-bar visible in full-screen mode. I have a zillion tabs open, most of them coding related, switching across them is made easier by the side-bar being visible.

Now I a bit frustrated, to switch tabs I have to either cycle through them with alt+tab, or exit full-screen and come back. 

I now search tabs can be used, but the shortcut `Cmd + Shift + A`, conflicts with in-browser slack.

**Comment by balhissay** [comment]
I would also love to be able to keep vertical tabs while on fullscreen in macOS. A specific option for this will be more than welcome. Thanks!!

**Comment by raduc** [comment]
+1, I always use Brave in full screen mode, especially on smaller screens. This update totally breaks my experience. At lease please add an option to return to the previous behavior.

**Comment by raduc** [comment]
Can the behavior be similar to horizontal tabs? They do show up in full screen mode, but then they hide when for example a video is playing full-screen. That would be the best behavior.

**Comment by hho** [comment]
@raduc Horizontal tabs don't show up in fullscreen for me either (Linux) – and I think that's correct: Full screen should be just that, a screen full of the webpage shown.

Do I understand you correctly, that you just want a maximized window minus the address bar?

**Comment by raduc** [comment]
Horizontal tabs do show up in MacOS full screen, see also https://github.com/brave/brave-browser/issues/33020
Some people prefer running the browser in full screen mode especially with smaller screen laptops (e.g. Macbook 12 inch).

**Comment by raduc** [comment]
On MacOS there is a difference between full-screen mode (tabs available) and presentation mode (tabs hidden):
https://apple.stackexchange.com/a/113736

**Comment by jvervlied** [comment]
I am adding to the requests of being ale to have vertical tabs available in fullscreen. I use brave in fullscreen exclusively and not having tabs visible makes it unusable.

**Comment by Blufords** [comment]
Technically the original bug is fixed and a new issue should be made for the requesting the return of the old feature, right?

**Comment by phildufresne** [comment]
I would argue that this bug-fix has been incorrectly implemented and needs to be reworked with MacOS full-screen / presentation-mode concepts considered.

**Comment by phildufresne** [comment]
Update: it was reverted - https://github.com/brave/brave-browser/issues/33106

**Comment by yozachar** [comment]
```console
$ brave --version
Brave Browser 117.1.58.137
```

To reproduce:

1. Use Brave `117.1.58.137`.
2. Go to: https://piped.kavin.rocks/watch?v=uiTrB-zA7bc or any YT video link.
3. Hit <kbd>F</kbd> and  the video will go to full-screen **without** the vertical tabs. Hit <kbd>F</kbd> again to quit full-screen.
5. Hit <kbd>F11</kbd> (or <kbd>Fn</kbd> + <kbd>F11</kbd>) and the web-page will go to full-screen **with** the vertical tabs visible. Press same key combination again to exit full-screen.

https://github.com/brave/brave-browser/assets/38415384/3f97c809-a3aa-4f9f-860c-4346ee6f799a

The behaviors is a bit odd, as if <kbd>F11</kbd> is treating vertical tabs as part of the web-page.

**Comment by raduc** [comment]
This has regressed and it's happening again in 1.63.174.

**Comment by My-life-is-over** [comment]
Any update on this?
